integrations-and-supported-tools/pytorch/scripts/Neptune_Pytorch_Support_Advanced.py

```diff
@@ -0,0 +1,182 @@
+from numpy.random import permutation
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+import matplotlib.pyplot as plt
+from torchvision import datasets, models, transforms
+import neptune.new as neptune
+import numpy as np
+
+def main():
+    # Step 1: Initialize Neptune and create new run
+    run = neptune.init(project='common/pytorch-integration', api_token='ANONYMOUS', source_files=['*.py'])
```

Comment:
I think we could/should add a name to the run so that it is easy to distinguish between those different use cases

Reply:
How about tags?

Reply:
Done